sdk/azcore/arm/runtime/policy_bearer_token.go

```diff
@@ -28,12 +28,12 @@ type acquiringResourceState struct {
 	tenant string
 }
 
-// acquire acquires or updates the resource; only one
-// thread/goroutine at a time ever calls this function
-func acquire(state acquiringResourceState) (newResource azcore.AccessToken, newExpiration time.Time, err error) {
+// acquireAuxToken acquires a token from an auxiliary tenant. Only one thread/goroutine at a time ever calls this function.
+func acquireAuxToken(state acquiringResourceState) (newResource azcore.AccessToken, newExpiration time.Time, err error) {
 	tk, err := state.p.cred.GetToken(state.ctx, azpolicy.TokenRequestOptions{
-		Scopes:   state.p.scopes,
-		TenantID: state.tenant,
+		EnableCAE: true,
```

Comment:
To make sure I understand, when requesting a token from an auxiliary tenant, we always want CAE enabled even if it wasn't enabled for the client?

Reply:
Oh wait, `EnableCAE` isn't set at the client level.